src/python/pants/backend/project_info/dependencies.py

```diff
@@ -38,7 +34,20 @@ def register_options(cls, register):
             default=False,
             type=bool,
             help=(
-                "List all transitive dependencies. If unspecified, list direct dependencies only."
+                "Act on the transitive dependencies of the input targets. If unspecified, act "
+                "only on the direct dependencies of the input targets."
+            ),
+        )
+        register(
+            "--format",
```

Comment:
The `dependees` goal uses `--output-format`, would be good to align: https://github.com/pantsbuild/pants/blob/906099f5c0d2f9e6adcac23e030270c64ca519ae/src/python/pants/backend/project_info/dependees.py#L98-L126

Reply:
(or deprecate and shorten the name on `dependees`)

Reply:
I think format makes sense and is more ergonomic

Reply:
Hmm, maybe `--output-format` is better, to go with `--output-file` from the `Outputting` subsystem base class.

Reply:
OTOH `--format` is shorter. I'll make a game-time decision...